lib/active_job/job_proxy.rb

```diff
@@ -20,6 +20,10 @@ def serialize
     end
   end
 
+  def retries
+    self.arguments.first["executions"] || 0
```

Comment:
Hmmm... I have two questions about this... first, do we need `self`? Second, does this work in general? 🤔 This extends `ActiveJob::Base`, which means `arguments` are the deserialized job arguments (which probably won't be deserialized here yet, because they're lazily deserialized when they're needed).

Reply:
in fact the self was not necessary.
And yes, it works, at least it is working on the screens where I applied it lol